includes/amp-helper-functions.php

```diff
@@ -1421,7 +1432,7 @@ function amp_is_dev_mode() {
 	 * queries for the expressions returned by the 'amp_dev_mode_element_xpaths' filter.
 	 *
 	 * @since 1.3
-	 * @param bool Whether AMP dev mode is enabled.
+	 * @param bool $is_dev_mode_enabled Whether AMP dev mode is enabled.
```

Comment:
Also noticed the docs for the `amp_dev_mod_enabled` filter was being rendered as:

![image](https://user-images.githubusercontent.com/16200219/103162268-66f03300-47e6-11eb-8f09-5311fe1edb4f.png)

So I added a named parameter called `$is_dev_mode_enabled`, so it will now be rendered as:

![image](https://user-images.githubusercontent.com/16200219/103162282-81c2a780-47e6-11eb-8b43-0fd8724146cd.png)